config.py

```diff
@@ -45,12 +45,12 @@
 TRAINING_STEPS = 1e10
 CORE_LSTM_LAYERS = 2
 OBSERVATION_SIZE = 4665
-OBSERVATION_TIME_STEPS = 5
-OBSERVATION_TIME_STEP_INTERVAL = 4
+OBSERVATION_TIME_STEPS = 10
+OBSERVATION_TIME_STEP_INTERVAL = 1
 INPUT_SHAPE = np.array([OBSERVATION_SIZE])
 ACTION_CONCAT_SIZE = 10
 # ACTION_DIM = [10, 5, 9, 10, 7, 4, 7, 4]
-ACTION_DIM = 10
+ACTION_DIM = 54

```

Comment:
If you are changing action_dim, you are also going to need to change action_concat_size because that is used in building the model.

Reply:
Thks, updated